packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/inference/takeVestPastAfterAir.yml

```diff
@@ -1,7 +1,7 @@
 languageId: typescript
 command:
   version: 1
-  spokenForm: take vest past after air
+  spokenForm: take vest past air
```

Comment:
This one isn't correct; see payload

Reply:
In our migration we're just dropping before/after for actions where they don't make sense

Reply:
Interesting. We should prob change this test to use "end of". I'll file; I think it's out of scope here

Reply:
Should we just do this in our migrations instead? tale after > take end of.